includes/models/class-nps-survey.php

```diff
@@ -75,7 +84,8 @@ public static function from_block_attributes( $attributes ) {
 			$attributes['surveyId'],
 			$attributes['title'],
 			$attributes['ratingQuestion'],
-			$attributes['feedbackQuestion']
+			$attributes['feedbackQuestion'],
+			! empty( $attributes['sourceLink'] ) ? $attributes['sourceLink'] : ''
```

Comment:
Nit: I believe this could be simplified to `$attributes['sourceLink'] ?: ''`?

Reply:
I was just going with the linter:
![image](https://user-images.githubusercontent.com/157240/106623242-2c02bd00-6553-11eb-8a67-030f165a587f.png)

Reply:
Didn't realize that was illegal 🤣